src/core/Akka/IO/Udp.cs

```diff
@@ -76,6 +76,16 @@ public SocketConnected(SocketAsyncEventArgs eventArgs)
             }
         }
 
+        internal sealed class SocketFailed : SocketCompleted
+        {
+            public readonly SocketAsyncEventArgs EventArgs;
+
+            public SocketFailed(SocketAsyncEventArgs eventArgs)
```

Comment:
It just occured to me that it is bad to pass `SocketAsyncEventArgs` around like this because it actually contains the buffer being used by `Socket.Receive()`. Doing this can cause memory leak later on.

Reply:
This was just a copy from the similar implementations above. I didn't question the existing command implementations.